awslabs/idf-modules
# sagemaker studio module

*Description of changes:*
- add generic version of sagemaker studio module from ADDF
- upgrade to latest CDK
- publish generic seedfarmer module metadata
- update env variable conventions
- remove redundant SSM params (duplicated by module metadata)
- add a flag to enable custom SM project templates (False by default)
- add workflow to run code checks and pytest
- a few test cases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
